### DISCUSSION TITLE
What is supposed to happen during `prisma generate` when you use `PRISMA_QUERY_ENGINE_BINARY`?

### DISCUSSION THREAD

**Discussion by janpio** [prisma/prisma#7781] [Q&A]
Should it copy that file into the generated Client as the query engine to use? 
(If so, which file name does it use for that? The custom one, or does it rename it to the default name?)
Should it not copy the file and assume you will also have the env var set when using that generated client?
Should it copy the _default_ file to the Client instead and not the one you specified via the env var? Does the default file even exist in this situation?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
